posts/2015-04-20-mix-it-2015-pt1.md

```diff
@@ -0,0 +1,50 @@
+Intro
+
+## L'importance de la performance :
+
+[Paul Bakaus](https://twitter.com/pbakaus) a insisté sur l'importance des performances d'une application :
+
+**Temps de chargement : 1s**
+
+Votre application doit être prête à l'emploi en maximum 1 seconde dès le premier accès de l'utilisateur. Sinon vous commencez à perdre des utilisateurs. À partir de 3 secondes, vous perdez 40% des utilisateurs. Et au-delà de 10 secondes, il ne reste plus personne !
```

Comment:
`premier accès de l'utilisateur` or `accès du premier utilisateur` ? not the same meaning.